src/internal.c

```diff
@@ -7526,6 +7685,10 @@ static int DoUserAuthFailure(WOLFSSH* ssh,
         ret = SendUserAuthRequest(ssh, authType, 0);
     }
 
+    if ((ret == WS_SUCCESS) && (authType & WOLFSSH_USERAUTH_KEYBOARD)) {
+        //ret = SendUserAuthKeyboard(ssh, authType);
```

Comment:
Please don't use C++ comments.

Reply:
Doh! That shouldn't be there at all, I thought I had removed it already.